requests_ntlm/requests_ntlm.py

```diff
@@ -13,25 +13,34 @@ class HttpNtlmAuth(AuthBase):
     Supports pass-the-hash.
     """
 
-    def __init__(self, username, password, session=None):
+    def __init__(self, username, password, session=None, domain=None):
         """Create an authentication handler for NTLM over HTTP.
```

Comment:
Prefacing this with `r` to make it a raw string will get rid of one set of escaping... Have to do this all the time in Ansible module docs for Windows paths...